### ISSUE TITLE
Update install instructions for conda-forge 

### ISSUE THREAD

**Issue by jthorton** [ACEsuit/mace#313]
**Is your feature request related to a problem? Please describe.**
Now that mace is available with all dependencies via conda-forge as `pymace` it would be great to have the install instructions also updated!

**Comment by ilyes319** [comment]
Yes we should do that, I am wondering why the name is pymace @davkovacs, I thought we were going for mace-torch just like pypi.

**Comment by jthorton** [comment]
It looks like someone from the community made the conda-package, is the plan to also create `mace-torch` or should we just reach out and also become maintainers of `pymace` and promote that installation pathway?

**Comment by gabor1** [comment]
it would be better to keep mace-torch as the name

**Comment by ilyes319** [comment]
We should make it `mace-torch` and ask the person making `pymace` to remove the package.

**Comment by jthorton** [comment]
Okay you should be able to copy the recipe of `pymace` and change the name to `mace-torch` and add you might need to add something which does not allow people to install them both at the same time as well.
